<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0.2441, y1:0.318, x2:0.263682, y2:1, stop:0.079602 rgba(84, 233, 255, 255), stop:0.636816 rgba(33, 194, 255, 255), stop:1 rgba(14, 179, 255, 255));}</string>
   </property>
   <widget class="QPushButton" name="buscarBoton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(254, 213, 27);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(37, 70, 180);

</string>
    </property>
    <property name="text">
     <string>Buscar Producto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inventarioBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(254, 213, 27);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(37, 70, 180);

</string>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agregarBoton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>420</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
text-align: center;
border-radius:20px;
background-color: rgb(254, 213, 27);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(37, 70, 180);
</string>
    </property>
    <property name="text">
     <string>Agregar Nuevos 
  Productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrarBoton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
text-align: center;
border-radius:20px;
background-color: rgb(254, 213, 27);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(37, 70, 180);
</string>
    </property>
    <property name="text">
     <string>Borrar 
Producto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>721</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 48pt &quot;Cascadia Mono SemiBold&quot;;
color: rgb(84, 158, 255);
</string>
    </property>
    <property name="text">
     <string>Menú Inventario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atrasBoton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>690</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border-radius:20px;
background-color: rgb(255, 255, 255);

font: 15pt &quot;MS Shell Dlg 2&quot;;
color: rgb(37, 70, 180);

</string>
    </property>
    <property name="text">
     <string>Atrás</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
